samples/csharp_dotnetcore/06.using-cards/Dialogs/MainDialog.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+#pragma warning disable CA1051 //Do not declare visible instance fields
```

Comment:
do we need thiis?, can we just change the `protected readonly ILogger _logger;` to private?